src/home/homed-home.c

```diff
@@ -2593,6 +2593,11 @@ static int on_home_ref_eof(sd_event_source *s, int fd, uint32_t revents, void *u
         if (h->ref_event_source_dont_suspend || h->ref_event_source_please_suspend)
                 return 0;
 
+        if (home_get_state(h) == HOME_INACTIVE) {
+                log_info("Got notification that all sessions of user %s ended, but it corresponds to the immediate state and there may be conflicting operations enqued already.", h->user_name);
```

Comment:
i'd downgrade this to log_debug(). This is interesting info for sure, but probably not something that people should really be pushed to read if everything operates cleanly